Mlem/App/Views/Shared/CommentView.swift

```diff
@@ -39,6 +39,7 @@ struct CommentView: View {
                 }
                 if !collapsed {
                     Markdown(comment.content, configuration: .default)
+                        .font(.callout)
```

Comment:
Did we decide to stick with `.body` here?

```suggestion
```

Reply:
We did, that was there from when I was grabbing comparison screenshots 🤦